### PR TITLE
feat(admin-email): add Email Recipients section to donor emails 2657

### LINKED ISSUE #2657: feat(admin-email): add Email Recipients section to donor emails
## Issue Overview
The edit email screen shows the "Email Recipients" section only on emails that can customize the recipients. See here (the New Donation email edit screen):
image

I think it would be beneficial to add that section to the emails that are sent to donors and that can't be customized, but with the setting greyed out and with a tooltip or explanation. Like so:

image

The input field doesn't allow typing into it at all, and the description reads:
> This email is sent to the Donor and the recipients cannot be customized.

This will improve the user-experience of this screen so they understand on-screen, without having to go elsewhere, where the email is sent, and not wonder why they can't add recipients to this email.

### PR DESCRIPTION
**Pull request by raftaar1191**
## Description
PR to fix #2657 

## How Has This Been Tested?
Manually tested by Making Donation

- [x] Tested By making an offline donation
- [x] Tested by making a test donation and check for a donation receipt 
- [x] Tested by creating a new donor and check for donor registration email
- [x] Tested by enabling the Email access option and then try to access it 

## Screenshots (jpeg or gifs if applicable):
image

image

## Types of changes
New feature (non-breaking change which adds functionality)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: includes/admin/emails/class-email-setting-field.php [hunk 1]
```diff
@@ -327,6 +329,35 @@ public static function get_recipient_setting_field( Give_Email_Notification $ema
 		return $recipient;
 	}
 
+	/**
+	 * Get recipient setting field that is disable.
+	 *
+	 * @since  2.0.7
+	 *
+	 * @access static
```
Comment (ravinderk) [inline includes/admin/emails/class-email-setting-field.php]: @raftaar1191 this should be `pubic`

File: includes/admin/emails/class-email-setting-field.php [hunk 2]
```diff
@@ -327,6 +329,35 @@ public static function get_recipient_setting_field( Give_Email_Notification $ema
...
+	public static function get_disable_recipient_setting_field( Give_Email_Notification $email, $form_id = null ) {
+		$recipient = array(
+			'id'         => self::get_prefix( $email, $form_id ) . 'disable_recipient',
+			'name'       => esc_html__( 'Email Recipients', 'give' ),
+			'desc'       => esc_html__( 'This email is sent to the Donor and the recipients cannot be customized.', 'give' ),
+			'default'    => '{donor_email}',
+			'value'      => '{donor_email}',
+			'type'       => 'text',
+			'attributes' => array(
+				'disabled' => true,
```
Comment (ravinderk) [inline includes/admin/emails/class-email-setting-field.php]: @raftaar1191 `disabled` must be set to `disabled` instead of `true`

**Comment by raftaar1191** [comment]
Tested this PR again @ravinderk it's working great

**Comment by DevinWalker** [comment]
@raftaar1191 please check the list off before adding to the "Ready for Review" pipeline. @ravinderk please review your last change request and let me know if this is good to go. Looks good from my end, I just want to be sure it's tested per form as well.

**Comment by raftaar1191** [comment]
@DevinWalker @ravinderk 

Tested this PR with Give Add-on as well like [Email Report](https://github.com/WordImpress/Give-Email-Reports)  and also with [Give Recurring ](https://github.com/WordImpress/Give-Recurring-Donations/)

Have found some issues in this two Add-on so I have sent PR to fixed it and after merge this two PR it's working great

**Pull request merged**